var/spack/repos/builtin/packages/p4est/package.py

```diff
@@ -10,13 +10,16 @@ class P4est(AutotoolsPackage):
     """Dynamic management of a collection (a forest) of adaptive octrees in
     parallel"""
     homepage = "http://www.p4est.org"
```

Comment:
Confirmed the following works and we prefer `https` over `http`:

```suggestion
    homepage = "https://www.p4est.org"
```